R/pkg/inst/tests/test_rdd.R

```diff
@@ -141,7 +141,7 @@ test_that("PipelinedRDD support actions: cache(), persist(), unpersist(), checkp
   unpersist(rdd2)
   expect_false(rdd2@env$isCached)
 
-  setCheckpointDir(sc, "checkpoints")
+  setCheckpointDir(sc, "checkpoint")
```

Comment:
This shouldn't be a directory in the project root to begin with. Make this a temp directory?